target/linux/airoha/dts/en7581.dtsi

```diff
@@ -227,6 +227,11 @@
 			      <0x0 0x1fbf0268 0x0 0x4>,
 			      <0x0 0x1fa2001c 0x0 0x50>,
 			      <0x0 0x1fa2018c 0x0 0x4>,
+			      <0x0 0x1fbf0208 0x0 0x4>, /* Interrupt status register */
+			      <0x0 0x1fbf0210 0x0 0x4>, /* Level 0 */
+			      <0x0 0x1fbf028C 0x0 0x4>, /* Level 1 */
+			      <0x0 0x1fbf020C 0x0 0x4>, /* Edge 0 */
+			      <0x0 0x1fbf0280 0x0 0x4>, /* Edge 1 */
```

Comment:
@merbanan IIUC we are missing irq support for bank1. Can you please add it as well and then we can see how to organize the code?

Reply:
Added.